__tests__/app.test.js

```diff
@@ -154,4 +154,46 @@ describe('/api/articles/:article_id/comments', () => {
             expect(body.error.message).toBe('Bad Request: Invalid article id')
         })
     })
+})
+
+describe('POST /api/articles/:article_id/comments',() => {
+test('Returns the posted comment', () => {
+    const newComment = {
+        username: 'lurker',
+        body: 'This is where the fun begins!'
+    }
+    return request(app)
+    .post('/api/articles/3/comments')
+    .send(newComment)
+    .expect(201)
+    .then(({body}) => {
+        const {comment} = body
+        expect(comment).toEqual({
+            comment_id: expect.any(Number),
+            body: 'This is where the fun begins!',
+            article_id: 3,
+            author: 'lurker',
+            votes: 0,
+            created_at: expect.any(String)
+        })
+    })
```

Comment:
We could add a test where the status is 201 although the body has additional properties that should be ignored 🙂